apps/products-feed/src/modules/google-feed/generate-google-xml-feed.ts

```diff
@@ -2,12 +2,12 @@ import { XMLBuilder } from "fast-xml-parser";
 import { GoogleFeedProductVariantFragment } from "../../../generated/graphql";
 import { productToProxy } from "./product-to-proxy";
 import { shopDetailsToProxy } from "./shop-details-to-proxy";
-import { EditorJsPlaintextRenderer } from "../editor-js/editor-js-plaintext-renderer";
 import { RootConfig } from "../app-configuration/app-config";
 import { getMappedAttributes } from "./attribute-mapping";
 import { priceMapping } from "./price-mapping";
 import { renderHandlebarsTemplate } from "../handlebarsTemplates/render-handlebars-template";
 import { transformTemplateFormat } from "../handlebarsTemplates/transform-template-format";
+import { EditorJsPlaintextRenderer } from "@saleor/apps-shared";
```

Comment:
we need to check if this gets tree-shaked, to ensure we don't import entire apps-shared every time

non blocking for now, just FYI to think about